assets/js/blocks/cart-checkout/cart/full-cart/style.scss

```
@@ -272,7 +272,7 @@ table.wc-block-cart-items {
 
 // Mobile styles.
 @include breakpoint( "<782px" ) {
-	.wc-block-cart {
+	.wp-block-woocommerce-cart > .wc-block-cart {
```

@mikejolley I added this to ensure the loading skeleton hid on mobile, there might be a more elegant way to finesse that.

---

You could also do:
```suggestion
	.wc-block-cart:not(.wc-block-cart--is-loading):not(.wc-block-cart--skeleton)
```
Doesn't make a big difference but I think it makes it easier to understand you want to select `.wc-block-cart` but not the loading skeleton.

---

Thanks, I think I prefer that, will try it before merge :)